earth_enterprise/BUILD_RPMS.md

```diff
@@ -47,13 +47,20 @@ allowed Open GEE Fusion and Server to be separately installed when building out
 of the Git repo checkout directly.
 
 ## Installing RPMs
-After the SCons **package_install** target completes you can install and
-test the RPMs you have created on CentOS or RHEL.  To do this, change to the
-*rpms/build/distributions* directory.  You can then use ```rpm -Uhv
-opengee-postgis-*.rpm``` and ```rpm -Uhv opengee-common-*.rpm``` to get the
-base dependencies installed, in that order.  You can then install ```rpm -Uhv
-opengee-fusion-*.rpm``` and/or ```rpm -Uhv opengee-server-*.rpm``` as desired,
-and then you can test the RPM packaged Open GEE Fusion and Server.
+After the SCons **package_install** target completes you can install and test
+the RPMs you have created on CentOS or RHEL.  To do this, change to the
+*rpms/build/distributions* directory.  You can then use ```sudo rpm -Uhv
+opengee-postgis-*.rpm``` and ```sudo rpm -Uhv opengee-common-*.rpm``` to get
+the base dependencies installed, in that order.  You can then install ```sudo
+rpm -Uhv opengee-fusion-*.rpm```, and then ```sudo rpm -Uhv
+opengee-server-*.rpm```, to test the RPM packaged Open GEE Fusion and Server.
+
+You can also use yum to install your RPM files, which will also automatically
+install standard dependencies for you.  However, since the opengee packages are
+not in a repository, using ```sudo yum install opengee-server-*.rpm``` or
```

Comment:
Couldn't you just do `sudo yum install opengee-*.rpm` ?

Reply:
Yea, you could, if you do want server and fusion and everything all at once...this was written originally from the perspective of installing server and fusion separately.